ts-redis/README.md

```diff
@@ -0,0 +1,39 @@
+# ORM Sample App
+
+## Run the app locally
+
+## Compile and Deploy with Cloudcc
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho ./dist --app sample-ts-redis
```

Comment:
Note: this command won't work for everyone because it creates an s3 bucket based on the app name, and those need to be globally unique.

Reply:
i can add $USER in there or something